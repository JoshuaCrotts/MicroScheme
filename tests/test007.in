;
; Non-tail-recursive factorial with cond. Larger number.
;

(define (fac n)
    (cond   [(= n 0) 1]
            [else (* n (fac (- n 1)))]
    )
)

(fac 80)
