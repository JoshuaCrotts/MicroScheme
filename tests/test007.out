7.156945704626378E118
